src/main/java/seedu/address/model/item/Resume.java

```diff
@@ -18,6 +18,7 @@
     // Item-level fields
 
     // Data fields
+    // Would it make sense to change this to internshipIds, or something like that? To be clear what these are
     private List<Integer> internships = new ArrayList<>();
     private List<Integer> projects = new ArrayList<>();
     private List<Integer> skills = new ArrayList<>();
```

Comment:
I need some variable naming advice here hahaha

Reply:
the internshipIds naming convention looks good.

Reply:
good foresight lol 👍

Reply:
Okay I will work on this!